dev-util/vulkan-headers/vulkan-headers-1.2.133.ebuild

```diff
@@ -3,29 +3,26 @@
 
 EAPI=7
 
-inherit cmake-utils
+MY_PN=Vulkan-Headers
```

Comment:
Don't we need to revbump all the 1.33 versions? It looks like only some are bumped.

Reply:
only those with manifest change were bumped.

Reply:
Sorry, makes sense